HelpSource/Classes/NamedControl.schelp

```diff
@@ -58,8 +58,12 @@ a.setn(\freq, [0, 2, 5].midiratio * 400);
 a = { SinOsc.ar(\freq.kr([300, 330, 370], [1, 0.3, 0.02])).sum * 0.1 }.play;
 
 // multiple usage of the same name:
-a = { SinOsc.ar(\freq.kr(440, 3.5)) + Saw.ar(\freq.kr(440, 0.05) * 0.5) * 0.1 }.play;
-
+(
```

Comment:
The example is indeed better code, but doesn't fulfil its purpose to demonstrate the feature.